packages/perseus/src/index.ts

```diff
@@ -190,7 +207,7 @@ export type {
     SharedRendererProps,
 } from "./types";
 export type {ParsedValue} from "./util";
-export type {Result, Success, Failure} from "./util/parse-perseus-json/result";
+export type {Result, Success, Failure} from "../../perseus-core/src/parse-perseus-json/result";
```

Comment:
I think this might have been a mistake from your IDE auto-updating imports. We shouldn't be importing from other packages this way. I think our linter for this is broken.